website/docs/d/organizations.html.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_organizations"
+sidebar_current: "docs-datasource-tfe-organizations"
+description: |-
+  Get information on Organizations.
+---
+
+# Data Source: tfe_organizations
+
+Use this data source to get a list of Organizations and a map of their IDs.
+
+## Example Usage
+
+```hcl
+data "tfe_organizations" "foo" {
+}
+```
+
+## Argument Reference
+
+No arguments are required. This retrieves the names and IDs of all the organizations.
```

Comment:
Maybe hint here that it may not be _all_ the organizations.

```suggestion
No arguments are required. This retrieves the names and IDs of all the organizations readable by the provided token.
```